timer/src/main/java/com/toggl/timer/log/domain/TimeEntriesLogAction.kt

```diff
@@ -39,4 +40,5 @@ fun TimeEntriesLogAction.formatForDebug() =
         is TimeEntriesLogAction.TimeEntryStarted -> "Time entry started $startedTimeEntry"
         is TimeEntriesLogAction.TimeEntryDeleted -> "Deleted time entry with id ${deletedTimeEntry.id}"
         is TimeEntriesLogAction.ToggleTimeEntryGroupTapped -> "Time entry group with id $groupId toggled"
+        is TimeEntriesLogAction.CommitDeletion -> "Commit deletion of $ids"
```

Comment:
Use the past tense here
```suggestion
        is TimeEntriesLogAction.CommitDeletion -> "Commited deletion of time entries with ids $ids"
```